lib/utils.pm

```diff
@@ -886,6 +887,29 @@ sub ensure_serialdev_permissions {
     }
 }
 
+=head2 disable_serial_getty
+Serial getty service pollutes serial output with login propmt, which
+interferes with the output, e.g. when calling script_output.
+Login prompt messages on serial are used on some remote backend to
+identify that system has been booted, so do not mask on non-qemu backends
+=cut
+sub disable_serial_getty {
+    my ($self) = @_;
+    my $service_name = "stop serial-getty\@$testapi::serialdev";
```

Comment:
I noticed this issue in my local run: http://dhcp126.suse.cz/tests/12124#step/stack_initialize/17

Reply:
Ouch, that's obviuously is a disgraceful typo, will fix that